### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] UI elements type for all the UIs should be communicated to the user

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11250]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
UI elements type for all the UIs should be communicated to the user.
Providing all the information about the UI on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.
UI created by NGX core and Platform, Not able to read its UI type like its Tiles.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader easily can check NGX core and platform component.
#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more context, specific examples, and what your expectation would be?

**Comment by JyotiKumari1507** [comment]
@droshev While testing Accessibility found that Screen reader isn't able to find type of component being used. Ex: If i use tiles with header 'Key Management' then Screen Reader is only able to recognise it as 'Key Management' . As per accessibility team Screen reader should able to read its UI type as 'Tiles' also. We can have more discussion about this in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev Attaching screen reader screenshot for more details:
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/5de44eda-b53a-44b1-b5f0-88e6b7db2bd5">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 any update?

**Comment by divyamalhotra488** [comment]
Hi @droshev, please find stackblitz example: https://stackblitz.com/edit/afrcak?file=src%2Fapp%2Fclickable-example.component.html

**Comment by mikerodonnell89** [comment]
This specific example is something that would be best handled by the application developer, in the event that they will want to customize the announcement to their needs. Also, from our library's perspective, each tile is unaware of the existence of any other given tile, so it does not make sense for any sort of label, description or announcement to come from fundamental-ngx

https://stackblitz.com/edit/afrcak-ihbrfr?file=src%2Fapp%2Fclickable-example.component.html%3AL11

<img width="1020" alt="Screenshot 2024-12-04 at 1 49 18 PM" src="https://github.com/user-attachments/assets/03406c0c-418f-48a0-abf6-00a3400f715b">
